oeps/best-practices/oep-0049-django-app-patterns.rst

```diff
@@ -267,7 +267,36 @@ Do note that even if you expose your tasks through ``api.py`` to be used by othe
 still be configured with the ``tasks`` import name, as the celery identifier for your task (as set by the celery
 decorator) is based off the original file.
 
+docs/decisions/0001-purpose-of-this-app.rst
+===========================================
+
+This should be an architectural decision record (ADR) describing the decision behind adding the app. Future ADRs should also be placed in this directory. See more about what should go in ADRs in the `ADRs section of OEP-0019`_
+
+If this is the only app in the repository, this ADR should just be a stub linking to the full ADR in `0001-purpose-of-this-repo.rst` (see `TODOs after running cookiecutter`_).
```

Comment:
Rather than adding this to the post-cookiecutter TODO list, could we add it to the cookiecutters as a part of this OEP change?

Reply:
As in postpone this update until we have updated the cookiecutter?

Reply:
Yeah, or at least until we have a draft PR ready to go for updating the cookiecutter. Since we are asking folks to make these additional ADRs starting now, my intuition is that it'd be best to provide the cookiecutter scaffolding now as well.

I don't know, maybe I'm underestimating the effort it'd take to update the cookiecutter. Don't take this as a blocking suggestion; what you have here is OK with me.

Reply:
I'm nervous about blocking this, so I'll leave this as is and then make an issue to add this to the cookiecutter if there isn't one already. If there is one I'll add a task to update this line.

Reply:
This is now https://github.com/openedx/edx-cookiecutters/issues/299. We can further discuss there.